WcaOnRails/db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
```

Comment:
I'm not sure what to do.

I fetched from upstream and then reset --hard upstream/master before making
local changes.

Should I fetch again and reset --soft upstream/master?

On Tue, Jan 5, 2016 at 8:54 AM, Tim Habermaas <notifications@github.com>
wrote:

> In WcaOnRails/db/structure.sql
> <https://github.com/cubing/worldcubeassociation.org/pull/298#discussion_r48833285>
> :
>
> > @@ -1,8 +1,8 @@
> > --- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
> > +-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
>
> git reset --soft HEAD~1 gets you in the state right *before* the commit
> (all changes are staged and git status tells you how to remove files from
> being staged).
>
> For the future you can use git add -p before commiting. This lets you
> step through every single change and asks you whether you want to stage it
> or not. Great feature.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/298/files#r48833285>
> .
>